UI/src/components/molecules/RipaStopResult.vue

```diff
@@ -561,6 +563,15 @@
                 Are you sure you want to select 'Contacted U.S. Department of
                 Homeland Security?'
               </ripa-alert>
+              <ripa-confirm-dialog
```

Comment:
You can move this outside of the conditional template and only write it once since the components are identical.